InvenTree/build/models.py

```diff
@@ -1049,6 +1050,38 @@ def after_save_build(sender, instance: Build, created: bool, **kwargs):
         # Run checks on required parts
         InvenTree.tasks.offload_task(build_tasks.check_build_stock, instance)
 
+        # Notify the responsible users that the build order has been created
+        if instance.responsible is not None:
+
+            targets = []
+
+            for owner in instance.responsible.get_related_owners(include_group=False):
+                user = owner.owner
+
+                if user != instance.issued_by:
+                    targets.append(user)
```

Comment:
Suggestion: this feels like something we need more often. I think it would warrant a change to get_related_user to exclude the supplied user(s).

Reply:
I have refactored this out, the `trigger_notification` method now accepts a set of users to exclude from the notification, which makes the calling code here a lot simpler.